### DISCUSSION TITLE
Revolt HypeSquad

### DISCUSSION THREAD

**Discussion by aeristhy** [stoatchat/discussions#223] [General]
HypeSquad in Revolt, only the hype of the hype can join. Not the current HypSquad houses in Discord but the old [HypeSquad](https://the-anime-god.is-inside.me/XilqIYtH.png)

**Comment by erkinalp** [comment]
HypeSquad is Discord Inc.'s marketing campaign. It is not related to Revolt in any manner.

**Reply by Rexogamer** [reply]
They are referring to revolt.chat (the platform), as opposed to the actual Revolt software. That being said, how exactly would a Revolt HypeSquad-esque system work?

**Comment by 4444dogs** [comment]
It should be called SecuritySquad
